internal/tiltfile/local_resource.go

```diff
@@ -83,6 +84,8 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 		"env?", &updateEnv,
 		"serve_env?", &serveEnv,
 		"readiness_probe?", &readinessProbe,
+		"cmd_dir?", &updateCmdDirVal,
+		"serve_cmd_dir?", &serveCmdDirVal,
```

Comment:
For consistency with `env` + `serve_env`, I think maybe we should name the actual Tiltfile params `dir` + `serve_dir`.

(FWIW The `_bat` ones are sort of different/special because they're actually _variations_ of the `cmd` value itself to use on Windows vs *nix/macOS. Of course, that said, both `env` + `dir` *are* actually part of the command, so arguably what you picked is more precise, buuut I think consistency/conciseness might be better since I don't think they're otherwise super ambiguous.)

Reply:
That makes sense -- I'll change it